enterprise/__init__.py

```diff
@@ -2,6 +2,6 @@
 Your project description goes here.
 """
 
-__version__ = "3.4.26"
+__version__ = "3.4.29"
```

Comment:
Will update the version numbers and changelog to what they should actually be. I had to do this to make sure that my local lms was using my updated code rather than using the cached, older version.

Reply:
I'd suggest going ahead and pulling in master and updating the version number from there